Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -917,7 +808,7 @@ bool RenderTheme::paint(const RenderBox& box, ControlStates& controlStates, cons
     case StyleAppearance::DefaultButton:
     case StyleAppearance::Button:
         return paintButton(box, paintInfo, integralSnappedRect);
-#endif
+#endif // USE(THEME_ADWAITA)
```

Comment:
I think we should leave this comment as `!USE(THEME_ADWAITA)`. It made for a very confusing review, as the diff makes it look like `paintCheckbox` is elided on non-Adwaita platforms, and some of the `StyleAppearance` values show up twice.